packages/shared/src/components/cards/squad/UnfeaturedSquadGrid.tsx

```diff
@@ -24,12 +24,7 @@ export const UnfeaturedSquadGrid = ({
   const title = source.name;
 
   return (
-    <Card
-      className={classNames(
-        'w-[19rem] overflow-hidden border-0 p-4',
-        className,
-      )}
-    >
+    <Card className={classNames('overflow-hidden border-0 p-4', className)}>
```

Comment:
I doubt it's actually needed though..

Reply:
I know, but since the card for featured also has a fixed width less that what it already was, but that is already an existing card I'm assuming this is just a glitch in design. 

Basing my argument on this note mentioned by Tsahi https://dailydotdev.slack.com/archives/C02E2C3C13R/p1725895066942129, and at the moment we have nothing pointing to the featured card that it is supposed to be changing. 

I'll ask to make sure, either way in this case we need to change both cards, either fixed width on both or not.

Reply:
Question here: https://dailydotdev.slack.com/archives/C069XNYMDGV/p1726755005672539